GradDemo.Api/Startup.cs

```diff
@@ -71,8 +71,11 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, Applicat
                 endpoints.MapControllers();
             });
 
-            if (!IntergrationTesting) 
-            db.Database.Migrate();
+            if (!IntergrationTesting)
+            {
+                //db.Database.Migrate();
```

Comment:
whats this?  Always check commits before pushing :)

Reply:
fixed